packages/sdk/react-client/src/client/ClientContext.tsx

```diff
@@ -99,15 +92,8 @@ export const ClientProvider = ({
   services: createServices,
   client: clientProvider,
   fallback: Fallback = () => null,
-  registerSignalFactory: register = true,
   onInitialized,
 }: ClientProviderProps) => {
-  useMemo(() => {
-    // TODO(wittjosiah): Ideally this should be imported asynchronosly because it is optional.
-    //   Unfortunately, aysnc import seemed to break signals React instrumentation.
-    register && registerSignalFactory();
```

Comment:
How is this not breaking?

Reply:
not sure, but i didnt mean to actually change it -- reverted